application/kitura/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ibmcom/kitura-ubuntu:latest
```

Comment:
It would be nicer if we had a more maintained and up to date base image for our example, but I couldn't find any alternatives.

This one has been last updated a year ago and only has the `latest` tag so it's not that nice.

It would be amazing if you can find a better base image. Otherwise I think we can have this as an example.

Reply:
Since Kitura is no longer developed (see https://en.wikipedia.org/wiki/Kitura), its final release having been one year ago, I think that shouldn't be much of a concern: https://github.com/IBM-Swift/Kitura/releases.
What do you think?